docs-gitbook/overview/how-ipc-compares.md

```diff
@@ -6,19 +6,29 @@ description: >-
 
 # How IPC compares
 
+## **Benefits of IPC Design**
+
+The IPC framework offers several significant benefits:
+
+- **Scalability**: By enabling the creation of subnets, IPC allows for on-demand horizontal scalability, effectively managing increased network load by distributing transactions across multiple chains.
+- **Flexibility**: The ability to tailor consensus mechanisms and stakeholder incentives per subnet caters to diverse application needs, optimizing performance and security.
```

Comment:
I think this is the direction we are going, but at the current stage, personally I feel it's not easy to switch consensus